Changelog.md

```diff
@@ -1,5 +1,10 @@
 # Spectrometer Changelog
 
+## v2.15.7
+
+- Resolve an issue where errors running `fossa report` and `fossa test` would be made more confusing when the project isn't a monorepo project ([#321](https://github.com/fossas/spectrometer/pull/321))
+- Prevent uploading standard analysis results to monorepo projects, where they'd be silently ignored ([#341](https://github.com/fossas/spectrometer/pull/341))
```

Comment:
[nit] we've been wording changelog entries as, e.g., "_Resolves_ an issue where ..." or "_Prevents_ ..."

It's as if the sentence implicitly starts like "Version v2.15.7 resolves an issue where ..." or "Version v2.15.7 prevents ..."